scripts/util/calc_global_phred_scores.py

```diff
@@ -232,7 +232,7 @@ def main(bam_path, prefix, fast_mode):
     parser.add_argument(
         "--fast_mode",
         action="store_true",
-        help="Only calculate the average, median, and standard deviation of the global PHRED scores",
+        help="Only calculate the first, middle, and last position PHRED scores isntead of also calculating PHRED scores for _every_ position",
```

Comment:
```suggestion
        help="Only calculate the first, middle, and last position PHRED scores instead of also calculating PHRED scores for _every_ position",
```